src/qt/pivx.qrc

```diff
@@ -59,6 +59,7 @@
         <file alias="explorer">res/icons/explorer.png</file>
         <file alias="automint_inactive">res/icons/automint_inactive.png</file>
         <file alias="onion">res/icons/onion.png</file>
+        <file alias="bitcoin_regtest">res/icons/bitcoin_regtest.png</file>
```

Comment:
change bitcoin_regtest to pivx_regtest

Reply:
I guess here the same answer applies: https://github.com/PIVX-Project/PIVX/pull/809#discussion_r254857699